scripts/linux_share_build.sh

```diff
@@ -56,19 +91,29 @@ function build_dagmc(){
                 -DBUILD_EXE=OFF \
                 -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
                 -DCMAKE_BUILD_TYPE=Release \
+                $CMAKE_ADDITIONAL_FLAGS \
                 -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/DAGMC
+    
     make -j$PROC
     make install
     cd ../..
     rm -rf DAGMC/DAGMC DAGMC/bld
 }
 
 function setup_Trelis_sdk() {
-    cd /Trelis-sdk 
-    dpkg -i Trelis-$1-Lin64.deb
 
+    cd ${FOLDER_PKG}
+    $SUDO apt install -y ./${TRELIS_PKG}
```

Comment:
Learning as I go here.. `apt` is meant for interactive package installs, providing colors, progress bar, etc. `apt-get` on the other hand is meant for use in scripts like one and can provide slightly different functionality than `apt`. Let's use `apt-get` in these files.

```suggestion
    $SUDO apt-get install -y ./${TRELIS_PKG}
```